src/components/StorageOverview/Alerts/utils.js

```diff
@@ -0,0 +1,3 @@
+import { get } from 'lodash';
+
+export const filterAlerts = alerts => alerts.filter(alert => get(alert, 'annotations.storage_type') === 'ceph');
```

Comment:
` get(alert, 'annotations.storage_type') ` should go to `selectors/prometheus/alerts.js`

Reply:
@suomiy I think it's specific to ceph cluster and will not be used for any other filtering. can you tell me why you think so?

Reply:
it is better to split the selectors from every place they are used in. So maybe to `selectors/prometheus/alerts/storage.js` ? and move also alerts.js to that folder ?

Reply:
ack! Got it. Thanks